jax_distribution/wrappers/jumanji.py

```diff
@@ -29,11 +29,11 @@ class LogEnvState:
     """State of the `LogWrapper`."""
 
     env_state: State
-    episode_returns: float
-    episode_lengths: int
+    episode_returns: jnp.float32
```

Comment:
maybe we can have a chat about this because we mainly did this to force having consistent type as skipping this caused us to have a function traced multiple time

Reply:
We didn't get an issue by changing the variables types to chex.Numeric however when we define it we need to keep it the float ones` jnp.float32(0.0)`